README.md

```diff
@@ -13,29 +17,48 @@ This project includes the following packages and configuration:
 * [Acquia platform memcache settings](https://github.com/acquia/memcache-settings)
 * Best practices for Drupal development, testing and project architecture
 
-The Acquia CMS starter kit allows you to install Drupal for a given style of CMS:
+The Acquia CMS Starterkit allows you to install Drupal for a given style of CMS:
 
 | Name  | Description |
 | ------------- | ------------- |
 | Acquia CMS Enterprise Low-code  | The low-code starter kit will install Acquia CMS with Site Studio and a UIkit. It provides drag and drop content authoring and low-code site building. An optional content model can be added in the installation process.  |
 | Acquia CMS Community  | The community starter kit will install Acquia CMS. An optional content model can be added in the installation process.  |
 | Acquia CMS Headless  | The headless starter kit preconfigures Drupal for serving structured, RESTful content to 3rd party content displays such as mobile apps, smart displays and frontend driven websites (e.g. React or Next.js).  |
 
-## Installation and usage
+For more information on selecting Acquia CMS Starterkit which best fit for
```

Comment:
For more information on selecting the Acquia CMS Starterkit that best fits your use case
 follow [instructions](https://docs.acquia.com/acquia-cms/install-cms)